### DISCUSSION TITLE
Spatial Data: DSM Generator ERROR

### DISCUSSION THREAD

**Discussion by VladimirSobral** [UMEP-dev/UMEP#578] [Q&A]
Hi

I have a problem when I try to generate a DSM of my ROI.
Can anyone help me, please?
Follow the DEM and Buildings files.

Thanks.
![image](https://github.com/UMEP-dev/UMEP/assets/124792761/40dcf1a8-dab3-46e0-b3c1-9d9decb35f98)
[DEM_AEROCLUBE.tif.aux.zip](https://github.com/UMEP-dev/UMEP/files/13675754/DEM_AEROCLUBE.tif.aux.zip)

**Comment by nilswallenberg** [comment]
Hi,

Your DEM is in a projected coordinate system (meters, EPSG:32725) and your shapefile in a geographic (degrees, EPSG:4326). They both need to be in a projected (meters/feet). Moreover, it looks like you are still in editing mode for your shapefile. Save your edits and stop editing. Then try to run the tool.

Hope this solves your issue. :)

Cheers,
Nils

**Reply by VladimirSobral** [reply]
Hi @nilswallenberg 

Thank you so much! I was able to run the DSM generator now!

Best,
Vlad

**Reply by nilswallenberg** [reply]
Great! :)
